src/Compilers/Test/Core/CompilerTraitAttribute.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Microsoft.CodeAnalysis.Test.Utilities
 {
-    [TraitDiscoverer("Microsoft.CodeAnalysis.Test.Utilities.CompilerTraitDiscoverer", assemblyName: "Roslyn.Test.Utilities")]
+    [TraitDiscoverer("Microsoft.CodeAnalysis.Test.Utilities.CompilerTraitDiscoverer", assemblyName: "Microsoft.CodeAnalysis.Test.Utilities")]
```

Comment:
This name being wrong broke the ability for `[CompilerTrait(...)]` to show up as a filter in the test explorer. That is a really handy feature with a change like this where I needed to run a bunch of lifetime tests across multiple test classes. Just put the right trait on them, run the tests and iterate.